datafusion/expr/src/logical_plan/plan.rs

```diff
@@ -1223,6 +1237,19 @@ pub struct DropTable {
     pub schema: DFSchemaRef,
 }
 
+/// Drops a schema
+#[derive(Clone, PartialEq, Eq, Hash)]
+pub struct DropSchema {
```

Comment:
Is this a pair with `CreateCatalogSchema`? If so, the name looks inconsistent.

Reply:
Good call thank you! 

I was modeling the name after `DropTable` and `DropView` but you are definitely right!